testing.md

```diff
@@ -0,0 +1,19 @@
+Testing
```

Comment:
Must be split into an other commit!

Reply:
The workflow is well documented and helps to shed some light on the wiki's shadows.
Thank you!

Reply:
My mistake, I didn't see the wiki at all! Will remove this file and instead update the wiki.

Reply:
I updated the [wiki](https://github.com/Lokicoule/nestjs-cognito/wiki) with the contents of your testing file.
Thank you kelly ;)